python/cudf/cudf/core/dataframe.py

```diff
@@ -4174,6 +4174,8 @@ def query(self, expr, local_dict={}):
                         datetimes
         1 2018-10-08T00:00:00.000
         """
+        if local_dir is None:
+            local_dir = {}
```

Comment:
We should move this under the `with annotate` context to make sure we capture it in profiling.

Reply:
oh sure, I can do that

Reply:
done in https://github.com/rapidsai/cudf/pull/6410/commits/da4cc1dc2a5183807e9437afc41f5625104050fd